<launch>

  <arg name="scale" default="700" />
  <arg name="port" default="4003" />
  <arg name="stdout" default="screen" />
  <arg name="net_name" default="didi" />


  <param name="/birdview/grid_dim" value="70" />
  <param name="/birdview/cell_size" value="0.1" />

   <node pkg="didi_challenge" type="smart_tracking" name="smart_tracking" output="screen">
    <param name="distance_threshold" value="40" />
    <param name="max_missings" value="100" />
    <param name="min_detections" value="2" />
  </node>
</launch>
